gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigV2ResourceLocalHandler.java

```diff
@@ -88,7 +101,7 @@ public CreateKVResponse createFlowConfig(FlowConfig flowConfig, boolean triggerL
           addSpecResponse != null && addSpecResponse.getValue() != null ? StringEscapeUtils.escapeJson(addSpecResponse.getValue()) : "");
       flowConfig.setProperties(props);
       httpStatus = HttpStatus.S_200_OK;
-    } else if (Boolean.parseBoolean(responseMap.getOrDefault(ServiceConfigKeys.COMPILATION_SUCCESSFUL, new AddSpecResponse<>("false")).getValue().toString())) {
+    } else if (Boolean.parseBoolean(response.getValue())) {
```

Comment:
because of the raw type, it may be slightly safer to keep the (seemingly superfluous) `.toString()` from the original.  in fact, you could possibly not add line 92 and preserve this one as in the orig.